include/picongpu/plugins/particleCalorimeter/ParticleCalorimeter.x.cpp

```diff
@@ -119,8 +119,8 @@ namespace picongpu
             {
                 const float_64 minEnergy_SI = this->minEnergy * sim.unit.energy();
                 const float_64 maxEnergy_SI = this->maxEnergy * sim.unit.energy();
-                const float_64 minEnergy_keV = minEnergy_SI * sim.si.conv.joule2ev(1.0e-3);
-                const float_64 maxEnergy_keV = maxEnergy_SI * sim.si.conv.joule2ev(1.0e-3);
+                const float_64 minEnergy_keV = minEnergy_SI * sim.si.conv().joule2ev(1.0e-3);
+                const float_64 maxEnergy_keV = maxEnergy_SI * sim.si.conv().joule2ev(1.0e-3);
```

Comment:
```suggestion
                float64 const keVperJ = sim.si.conv().joule2eV(1.0e-3);

                float_64 const minEnergy_keV = minEnergy_SI * keVperJ;
                float_64 const maxEnergy_keV = maxEnergy_SI * keVperJ;
```
still not really happy with but don't see a better version